model/AI/AI.md

```diff
@@ -4,18 +4,18 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Additional metadata based on software profile, that is useful for ai applications and models.
+The AI Profile provides additional metadata, based on Software Profile, that is useful for AI applications and models.
```

Comment:
@kestewart  @bact  Could we change summary AI profile definition to The AI profile is designed to provide a standardized way of documenting and sharing information about AI software packages (ie. system)

Reply:
I have updated the PR as proposed